observability/cockpit/how-to/manage-external-data-from-cockpit.mdx

```diff
@@ -0,0 +1,50 @@
+---
+meta: How to manage your external data from your Cockpit
```

Comment:
did you mean "How to send metrics and logs to your Cockpit" ?

(applies many times in this doc)